AutoDraft/arima_analysis.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+import matplotlib.pyplot as plt 
+import streamlit as st
+import numpy as np
+import matplotlib.pyplot as plt
+from datetime import datetime as dt
+from bokeh.layouts import gridplot, column
```

Comment:
nit: remove unused imports (you can get a linter in your IDE like flake8 to tell you about this)

Reply:
Linted all files (oof)!